### ISSUE TITLE
Post Agent seemingly always reports 'not working'

### ISSUE THREAD

**Issue by cskwrd** [huginn/huginn#3034]
I have configured a Post Agent to run periodically. However the working status always (I let it be for near 24 hours) shows `no`. From the events it emits, it seems to be running just fine.

I have never written ruby, but from the agent's working [function definition](https://github.com/huginn/huginn/blob/b06f77d1d78b08a82b9ca381702506403f35eb58/app/models/agents/post_agent.rb#L90), it reads as though it is never expected to be run on a schedule.

To me it seems that the agent only reports a positive working status if both of the following conditions are true.

1. No error logs exist. [Code](https://github.com/huginn/huginn/blob/b06f77d1d78b08a82b9ca381702506403f35eb58/app/models/agents/post_agent.rb#L91)
2. An event has been propagated to the Post Agent in fewer days than specified by the undocumented property `expected_receive_period_in_days`. [Code](https://github.com/huginn/huginn/blob/b06f77d1d78b08a82b9ca381702506403f35eb58/app/models/agents/post_agent.rb#L93-L95)

Am I reading that correctly? Should there be a check to validate if the agent is emitting events properly? Or maybe on the HTTP status code of the response to the request made by the agent?

For what it's worth I am using the latest (or near latest) `huginn/huginn` docker container.

**Comment by cskwrd** [comment]
I was looking at this a little more. Would it make sense that logic for the Post Agent and Website Agent would be similar? The [function definition](https://github.com/huginn/huginn/blob/b06f77d1d78b08a82b9ca381702506403f35eb58/app/models/agents/website_agent.rb#L221) for the Website Agent seems to be more like what I would expect.

**Comment by peacepenguin** [comment]
I have the same issue on my post agent to browserless, works great, but annoys me that shows "not working" and "working: no" in the console status.

So, i recently tried this: toggled view to code mode on the agent config and deleted the default line:
`
  "expected_receive_period_in_days": "1",
`
After saving the agent immediately switched to "working: yes"

Which makes sense, my post agent is the source of data for my flow, so it will never receive any events itself, so this line should be gone on this type of post agent.

**Comment by meminens** [comment]
If I remove expected days and save, it gives me error message saying that it is required and won't save.

**Comment by gingerbeardman** [comment]
Yeah, a Huginn update added  "expected_update_period_in_days": "1", back into my agents
